app/models/user.rb

```diff
@@ -47,5 +50,44 @@ def at_least_2_interests
       false
     end
   end
-# make test for phone, zip, and profile pic
+
+  def recommended_projects
+    ideal_project = self.projects.create(name: "ideal project",
+    proposal: "This is an ideal project generated by the user's skills and
+    interests and temporarily used for the purpose of finding releated
+    projects",
+    skill_list: self.skill_list.join(", "),
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Redundant `self` detected.